Pixel Game/Assets/ScenePrefabs/Canvas.prefab

```diff
@@ -6908,7 +6908,7 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 38.431, y: 0}
+  m_AnchoredPosition: {x: 36.5, y: -2.1}
```

Comment:
This i noticed for myself, ill change it back before we merge. Might have been a mistake

Reply:
What is this about?

Reply:
This is correct ```m_AnchoredPosition: {x: 38.431, y: 0}```